optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/pom.xml

```diff
@@ -164,6 +164,16 @@
       <artifactId>uberfire-services-backend</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-ssh-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-ssh-backend</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
@pefernan Pere, I don't think the solver editor backend should depend on UberFire SSH. Is there some non-obvious reason for that?

Reply:
@yurloc I tried the tests and there were some weld unsatisfied dependencies error, I had to add the deps to solve that.